src/test/resources/overseas_entity_v_3_1.json

```diff
@@ -504,6 +612,100 @@
         "resigned_on" : "2009-08-01",
         "ch_reference" : "MOcorp1"
       }
+    ],
```

Comment:
The model does not contain update object @AndreKramer7654 will that needed to be added in another ticket

Reply:
I'm confused. Isn't that structure present from line 327 of this file?(I think @markpit added this fairly recently, to be compliant). It's there as a field in the `OverseasEntitySubmissionDto` too.

Reply:
Seems I missed that ...